### ISSUE TITLE
VIS/BUG: subplots=True gives rotated tick labels

### ISSUE THREAD

**Issue by jorisvandenbossche** [pandas-dev/pandas#15025]
#### Problem description

When using the normal `plot()` method vs `plot(subplots=True)`, you get a different rotation for the ticklabels. I don't think this inconsistency is needed, and would use non-rotated ticklabels for `subplots=True` as well.

#### Code Sample

```
In [6]: df = pd.DataFrame(np.random.randn(20,2), columns=['A', 'B'], index=pd.date_range("2016-01-01", periods=20))

In [8]: df.plot()
Out[8]: <matplotlib.axes._subplots.AxesSubplot at 0x7f00389f0198>
```

![figure_1-nosubplots](https://cloud.githubusercontent.com/assets/1020496/21577135/6ea4f328-cf50-11e6-808b-28967c0ca1d9.png)

```
In [9]: df.plot(subplots=True)
Out[9]: 
array([<matplotlib.axes._subplots.AxesSubplot object at 0x7f00333381d0>,
       <matplotlib.axes._subplots.AxesSubplot object at 0x7f00332ddf98>], dtype=object)
```

![figure_1_subplots](https://cloud.githubusercontent.com/assets/1020496/21577140/8412ca50-cf50-11e6-9ff9-14d3ef3d97d9.png)

The above example is with a time series, but you actually get exactly the same when eg just using the default integer index.

Using pandas master, 0.19.0+289.g1bf94c8

**Comment by sinhrks** [comment]
Yeah, the default rotation is not needed as users can overwrite via ``rot`` kw.

**Comment by sun-0-set** [comment]
This looks like it was fixed by PR #34334, which closed #29460 (a duplicate of this issue).
